dashboard/test/unit/test_subscribe.js

```diff
@@ -130,7 +155,7 @@ suite('subscribe', function () {
 
         //when
         subscribe.isSubscribed(user, region, function () {
-            assert(!region.subscribed);
+//            assert(!region.subscribed);
```

Comment:
Is this assert commented on purpose? If so, maybe we should just remove the line